src/structures/interfaces/TextBasedChannel.js

```diff
@@ -99,7 +99,8 @@ class TextBasedChannel {
    * Options for splitting a message.
    * @typedef {Object} SplitOptions
    * @property {number} [maxLength=2000] Maximum character length per message piece
-   * @property {string} [char='\n'] Character to split the message with
+   * @property {string | string[] | RegExp | RegExp[]} [char='\n'] Character(s) or Regex(s) to split the message with,
```

Comment:
```suggestion
   * @property {string|string[]|RegExp|RegExp[]} [char='\n'] Character(s) or Regex(s) to split the message with,
```

Reply:
Consistency with the rest of the library, mostly.

Reply:
sure, just curious, added